apps/web/app/lib/surveys/surveys.ts

```diff
@@ -481,17 +505,23 @@ export const getFormattedFilters = (
   if (meta.length) {
     meta.forEach(({ filterType, questionType }) => {
       if (!filters.meta) filters.meta = {};
-      if (!filterType.filterComboBoxValue) return;
-      if (filterType.filterValue === "Equals") {
-        filters.meta[questionType.label ?? ""] = {
-          op: "equals",
-          value: filterType.filterComboBoxValue as string,
-        };
-      } else if (filterType.filterValue === "Not equals") {
-        filters.meta[questionType.label ?? ""] = {
-          op: "notEquals",
-          value: filterType.filterComboBoxValue as string,
-        };
+
+      // For text input cases (URL filtering)
+      if (typeof filterType.filterComboBoxValue === "string" && filterType.filterComboBoxValue.length > 0) {
```

Comment:
Could we make this check stricter by checking if `questionType.id === ‘url’`? We also get string values for select dropdowns, such as in the OS selector.

<img width="1148" height="410" alt="Image" src="https://github.com/user-attachments/assets/b1afa6e8-d9cd-4fd4-b3a4-bf72cdd572cb" />